test.cmd

```diff
@@ -0,0 +1,24 @@
+@echo off
+setlocal
+cd "%~dp0"
+chcp 1252 > nul
+if not exist tools\NUnit.Runners (
+    for %%i in (NuGet.exe) do set nuget=%%~dpnx$PATH:i
+    if "%nuget%"=="" goto :nonuget
+    md tools
+    nuget.exe install NUnit.Runners -Version 2.6.4 -OutputDirectory tools
+    move .\tools\NUnit.Runners.2.6.4 .\tools\NUnit.Runners
+)
+if not %errorlevel%==0 exit /b %errorlevel%
+
+call build /v:m
+.\tools\NUnit.Runners\tools\nunit-console.exe .\MoreLinq.Test\bin\Debug\net40\MoreLinq.Test.dll
```

Comment:
If building fails in the previous step then tests shouldn't be run. Would be better to predicate each on the success of the previous using `&&`, as in:

```cmd
call build /v:m ^
  && .\tools\NUnit.Runners\tools\nunit-console.exe .\MoreLinq.Test\bin\Debug\net40\MoreLinq.Test.dll ^
  && .\tools\NUnit.Runners\tools\nunit-console.exe .\MoreLinq.Test\bin\Release\net40\MoreLinq.Test.dll ^
  && .\tools\NUnit.Runners\tools\nunit-console.exe .\MoreLinq.Test\bin\Debug\net40-client\MoreLinq.Test.dll ^
  && .\tools\NUnit.Runners\tools\nunit-console.exe .\MoreLinq.Test\bin\Release\net40-client\MoreLinq.Test.dll
```